neural_compressor/config.py

```diff
@@ -521,9 +542,16 @@ def strategy(self):
     @strategy.setter
     def strategy(self, strategy):
         if check_value('strategy', strategy, str,
-            ['basic', 'mse', 'bayesian', 'random', 'exhaustive']):
+            ['basic', 'mse', 'bayesian', 'random', 'exhaustive', 'sigopt', 'tpe']):
```

Comment:
Please add `mse_v2` to the checklist.

Reply:
resolved:
1617594c52897d4f278905978f42e3be23e5e79d